src/ua/client.rs

```diff
@@ -67,6 +81,15 @@ impl Client {
 
 impl Drop for Client {
     fn drop(&mut self) {
+        log::info!("Disconnecting from endpoint");
+
+        let status_code = ua::StatusCode::new(unsafe { UA_Client_disconnect(self.as_mut_ptr()) });
```

Comment:
Maybe add a comment about the (non-)blocking behavior of `UA_Client_disconnect()` and a link? I couldn't find this information after a quick Google search.